src/renderer/views/Playlist/Playlist.js

```diff
@@ -418,6 +437,23 @@ export default defineComponent({
       showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist does not exist'))
     },
 
+    checkPlaylist: function () {
+      if (
+        this.sortOrder === SORT_BY_VALUES.VideoLengthAscending ||
+        this.sortOrder === SORT_BY_VALUES.VideoLengthDescending
+      ) {
+        if (!this.shownNotification) {
```

Comment:
Why need this

Reply:
It doesn't strictly need the `if` statement, but it will speed it up if it knows that the `sortOrder` doesn't care about the length of the videos, as then it doesn't have to iterate through all the videos in the playlist